diffus/src/lcs.rs

```diff
@@ -19,12 +22,20 @@ impl<T: Eq> Lcs<T> {
 
         let mut storage = vec![0; width * height];
 
+        let a = |i: usize, j: usize| {
```

Comment:
Do we want to use the `TwoDVec` equvalent in https://github.com/Jim-Holmstroem/diffus/commit/94959d9acd6c338d7688b18d850ff17f7ad1098f ? And also extract the type to a separate module

Reply:
yes in a separate PR :)
Its quite a bit of code and logic, that you can also solve with a little lambda a, judgment call I guess